tests/5_verification.ts

```diff
@@ -10,25 +10,24 @@ async function main() {
   // create an attested claim from the JSON string
   const attestedClaimStruct = (await setup()).attestedClaim // ❗️ REMOVE-LINE in workshop ❗️
   // const attestedClaimStruct = JSON.parse('<attestedClaimJSONString>') //❗️ UNCOMMENT-LINE in workshop ❗️
-  const attestedClaim = Kilt.AttestedClaim.fromAttestedClaim(
-    attestedClaimStruct
-  )
+  const attestedClaim =
+    Kilt.AttestedClaim.fromAttestedClaim(attestedClaimStruct)
 
-  await Kilt.default.connect(Kilt.BlockchainApiConnection.DEFAULT_WS_ADDRESS) // ❗️ REMOVE-LINE in workshop ❗️
-  // await Kilt.default.connect('ws://full-nodes.devnet.kilt.io:9944') // ❗️ UNCOMMENT-LINE in workshop ❗️
+  await Kilt.init({ address: 'wss://full-nodes.kilt.io:9944' })
```

Comment:
we are calling kilt.init in setup and have removed it from all the other test files but this one

Reply:
Must have missed it.